CMIP6_studies/Ouranos.yaml

```diff
@@ -0,0 +1,355 @@
+- key: Ouranos tas bias
+  doi: [in-house performance]
+  metric: &tasclim
+    name: tasclim
+    long_name: tas climatology vs ERA5 climatology
+    units: K
+    variables: tas
+    comment: >
+      We calculate the RMSE at each grid point over the selected the domain (relative to ERA5) for the period from 1981 to 2010 and then we do a spatial average.
+      All models and ERA5 are linearly interpolated on a regular 1°x1° grid.  
+      -----
+      Periods, domains can be changed in https://pavics.ouranos.ca/selection-de-modeles
+    best: 1.21
+    worst: 2.26
+  type: performance
+  spatial_scope: NAM-CORDEX
+  temporal_scope: Climatology
```

Comment:
Temporal scope refers to the seasonal or annual extent https://github.com/WCRP-CORDEX/cmip6-for-cordex/blob/e6056efdbb698d17fcbd412db61cd27b43243ca9/CMIP6_studies/README.md?plain=1#L83 In this case, it should be Annual